### DISCUSSION TITLE
where is table component?

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#13365] [Help] [answered]
Hello all,
when trying to add the table component I get the table grid.
Self hosted  v 2.22.12 
I can copy, past and modify an existing table component, don't know what will happen in new apps.
Thank you foe your help

**Comment by ConorWebb96** [comment, marked as answer]
Hey @JCV67,

The table has been deprecated. Going forward the Grid component will be the component used as the table. Older apps with the old flow should still work so theres no need to worry. 

There was an announcement made within the discord around this. One will be made within Discussions sometime today. You can find the related discord announcement [here](https://discord.com/channels/733030666647765003/1020065208552652852/1221810756417355806).

Hopefully this makes sense.

**Reply by JCV67** [reply]
I'm sorry to ear that. The grid has components thar are not good for end users table designs. Can't find a way to remove column number or gridlines, or even label icons.

**Reply by ConorWebb96** [reply]
Thanks for the feedback I'll forward these suggestions onto product so they can have a look!

**Reply by ConorWebb96** [reply]
Also just to address a few of your points you can toggle columns off that you don't want visible via the toggle in the below screenshot.

You can also amend the label by clicking into the settings of the fields displayed here and change the names. The only aspect you're missing is the grid lines I believe. 👍 

<img width="307" alt="Screenshot 2024-03-27 at 12 51 54" src="https://github.com/Budibase/budibase/assets/126772285/b0d66916-7b88-4ad9-aeec-9aa5ddd7ae71">

<img width="605" alt="Screenshot 2024-03-27 at 12 53 17" src="https://github.com/Budibase/budibase/assets/126772285/70c77e08-d156-4804-8249-1a83fecf5d5a">

Hopefully this helps.

**Comment by JCV67** [comment]
Thank you @ConorWebb96 by now I'll keep copying tables from other screens. With grid it is harder to keep tables small and smooth.
